Docs/sphinx_documentation/source/LinearSolvers.rst

```diff
@@ -848,4 +849,76 @@ An example (implemented in the ``MultiComponent`` tutorial) might be:
 
 See ``amrex-tutorials/ExampleCodes/LinearSolvers/MultiComponent`` for a complete working example.
 
-.. solver reuse
+
+Curl-Curl
+=========
+
+The curl-curl solver supports solving the linear system arising from the
+discretized form of
+
+.. math:: \nabla \times (\alpha \nabla \times \vec{E}) + \beta \vec{E} = \vec{f},
+
+where :math:`\vec{E}` and :math:`\vec{f}` are defined on grid edges,
```

Comment:
Does "on grid edges" mean the same thing as "on cell edges?" If so, I think it's better to use "cell" for consistency since when I see "grid" I think "box."